src/lib.rs

```diff
@@ -210,6 +217,8 @@ impl core::fmt::Display for TOTP {
 }
 
 #[cfg(all(feature = "gen_secret", not(feature = "otpauth")))]
+// because `Default` is implemented regardless of `otpauth` feature we don't specify it here
+#[cfg_attr(docsrs, doc(cfg(feature = "gen_secret")))]
```

Comment:
Is it possible to do `cfg(all(feature = "gen_secret", not(feature = "otpauth")))` in the `doc()`? Because another `cfg_attr` is present in the other impl of default, and I'm not sure how cargo doc behaves in that case

Reply:
It's possible, but cargo output this confusing message
```
Available on crate feature gen_secret and non-crate feature otpauth only.
```

Reply:
Ah, yeah okay, it's kind of confusing indeed